eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/processors/ContigReplacerProcessorTest.java

```diff
@@ -47,33 +47,37 @@ public void setUp() throws Exception {
 
     @Test
     public void convertVariantFromSeqNameToSeqName() throws Exception {
-        SubSnpNoHgvs input = new SubSnpNoHgvs(SS_ID, RS_ID, ALTERNATE_ALLELE, ASSEMBLY, "", "", SEQNAME_1, START,
-                                              SEQNAME_1, DbsnpVariantType.SNV, Orientation.FORWARD, Orientation.FORWARD,
-                                              Orientation.FORWARD, START, true, true, REFERENCE_ALLELE, null, TAXONOMY);
+        SubSnpNoHgvs input = new SubSnpNoHgvs(SS_ID, RS_ID, REFERENCE_ALLELE, ALTERNATE_ALLELE, ASSEMBLY, "", "",
```

Comment:
Try to write as much parameters in each line as possible (without exceed 121 characters). Applies when creating new `SubSnpNoHgvs` in this test class